Appraisals

```diff
@@ -6,12 +6,20 @@ def add_appraise_for(activerecord_version:, pg_version:)
   appraise "activerecord_#{version_to_label(activerecord_version)}_pg_#{version_to_label(pg_version)}" do
     gem "activerecord", activerecord_version, require: "active_record"
     gem "pg", pg_version
+
+    # The following used to be default gems and ceased to be in recent Ruby versions.
+    # There is no harm in requiring them regardless of Ruby version.
+    gem "base64"
+    gem "benchmark"
+    gem "bigdecimal"
+    gem "logger"
+    gem "mutex_m"
```

Comment:
Needed for Ruby 3.5 (and gets rid of a bunch of deprecation warnings with Ruby 3.4).